screens/TournamentCreation/__tests__/CreateTournamentScreen.test.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { render } from "@testing-library/react-native";
+import CreateTournamentScreen from "../CreateTournamentScreen";
+
+
+describe("CreateTournamentScreen", () => {
+    test("it renders header and name input", () => {
+        const { getByText, getByRole } = render(<CreateTournamentScreen />);
+        expect(getByText("Create Tournament")).toBeDefined();
+        expect(getByText("Tournament Name")).toBeDefined();
+        
+        // Create Button
+        expect(getByRole("button")).toBeDefined();
+    });
+})
```

Comment:
Example of a very simple test - literally just "if someone breaks this component, it should fail"

Reply:
Cool, yeah, I have no idea how tests work in js/ts so I'll have to look into what the standard stuff is mroe